frontend/javascripts/libs/react_hooks.ts

```diff
@@ -2,13 +2,15 @@ import { useState, useEffect, useRef } from "react";
 import { useLocation } from "react-router-dom";
 
 // Adapted from: https://usehooks.com/usePrevious/
-export function usePrevious<T>(value: T): T | null | undefined {
+export function usePrevious<T>(value: T, ignoreFalsyValues: boolean = false): T | null {
```

Comment:
Slightly pedantic, it's not ignoring falsy values, but only null and undefined, isn't it? Not sure what to call those :see_no_evil: Maybe `ignoreNullValues`, although it's also not entirely correct :shrug:

Reply:
Of course, you are right. I think the right term is "nullable" :) I'll rename it!

Reply:
Is it really? :grimacing: I always read “nullable” as “can be null _or_ have a value” (similar to an option type), while in this context it reads more like “is actually null (or undefined)“. I may be wrong, though

Reply:
:see_no_evil: One could name it explicitly,`ignoreNullAndUndefined`.

Reply:
You're right. I got a bit confused reading https://www.typescriptlang.org/docs/handbook/advanced-types.html#nullable-types which doesn't really define what a nullable type is, but @fm3's take is how I actually see it, too. I renamed it to `ignoreNullAndUndefined` :)